iree/hal/driver_registry.h

```diff
@@ -102,6 +102,16 @@ typedef struct iree_hal_driver_registry_s iree_hal_driver_registry_t;
 IREE_API_EXPORT iree_hal_driver_registry_t* IREE_API_CALL
 iree_hal_driver_registry_default();
 
+// Creates and initializes an empty device driver registry
```

Comment:
```suggestion
// Creates and initializes an empty device driver registry.
```

style nit: add period

(higher bar for comment style in API header files, sorry D:)